crates/ruma-events/src/room/message.rs

```diff
@@ -831,17 +833,35 @@ pub struct TextMessageEventContent {
     /// Formatted form of the message `body`.
     #[serde(flatten)]
     pub formatted: Option<FormattedBody>,
+
+    /// The extensible event form of the message.
+    #[cfg(feature = "unstable-msc1767")]
+    #[serde(flatten)]
+    pub message: Option<MessageContent>,
```

Comment:
Would be nice to come up with a more obvious name for this, but I don't have any ideas right now..